list.json

```diff
@@ -1,146 +1,286 @@
 {
 	"corpora": {
 		"quora-duplicate-questions": {
-			"description": "over 400,000 lines of potential question duplicate pairs. Each line contains IDs for each question in the pair, the full text for each question, and a binary value that indicates whether the line truly contains a duplicate pair.",
+			"num_rows": 404290,
```

Comment:
What are "rows"? We're not storing matrices, I don't see the connection to n-dimension axes. "Row" sounds too vague.

Reply:
Row it's one document / one object from the dataset, what name do you propose to use?

Reply:
Row is one document / object, OK, I'll rename it according to next comment

Reply:
Done 9c93d55